app/components/supplementary-file-browser/template.hbs

```diff
@@ -0,0 +1,65 @@
+{{#file-renderer download=selectedFile.links.download
+    width="99%" height="700"}}
+{{/file-renderer}}
+
+<div class="row">
+    <div class="col-sm-6">
+        <p class="f-w-lg p-b-md">{{selectedFile.name}}</p>
+    </div>
+    <div class="col-sm-6 text-nowrap supplemental-downloads">
+        {{#if (eq selectedFile.id preprint.primaryFile.id)}}
+            <a class="btn btn-default btn-sm" href={{fileDownloadURL}}> {{t "content.share.download_preprint"}}</a>
+        {{else}}
+            <a class="btn btn-default btn-sm" href={{fileDownloadURL}}> {{t "content.share.download_file"}} </a>
+        {{/if}}
+        <span class="p-sm"> {{t "content.version"}}: {{selectedFile.currentVersion}}</span>
+    </div>
+</div>
+
+{{#if hasAdditionalFiles}}
+    <section class="osf-box p-md m-t-sm m-b-lg">
+        {{#liquid-bind (slice-array files startIndex endIndex) use=scrollAnim as |list|}}
+            {{#each list as |supplement index|}}
+                {{#if (eq supplement.kind "folder")}}
+                    <a href={{concat node.links.html 'files'}} target="_blank" class="hint--bottom col-xs-2 p-md box m-b-xl hidden-xs" aria-label={{supplement.name}}>
+                        <i class="fa fa-folder fa-2x p-b-xs" aria-hidden="true"></i>
+                    </a>
+                    <a href={{preprint.links.html}} class="visible-xs truncate">
+                        <i class="fa fa-folder p-b-xs p-r-sm"></i> {{supplement.name}}
+                    </a>
+                {{else}}
+                    <label class="hint--bottom col-xs-2 p-md box m-b-xl hidden-xs {{if (eq selectedFile supplement) 'selected'}}" aria-label={{supplement.name}} {{action 'changeFile' supplement}}>
```

Comment:
Could this be a bootstrap tooltip? I don't think we've included hint.css in the project yet.